### DISCUSSION TITLE
在Xray中，使用Reality且设置TLS指纹为safari时，指纹是多少呀，怎么获取呢

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [XTLS/Xray-core#4077] [General]
在Xray中，使用Reality且设置TLS指纹为safari时，指纹是多少呀，怎么获取呢

**Comment by hsqStephenZhang** [comment]
[trackme](https://github.com/pagpeter/TrackMe), 搭建一下，curl 请求 /api/all 就能看到了.
可以参照我这里的测试代码 [main.go](https://gist.github.com/hsqStephenZhang/464792650322426c619ace61bdbc5ff5)

**Comment by xkdvSrPD** [comment]
![image](https://github.com/user-attachments/assets/6c280b5d-c227-49a2-95ac-925443e70913)
抓一个Client Hello的包就知道了（这不是你的正确答案）

**Reply by Twidora** [reply]
但是算下哈希就是了(雾
